docs/content/blog/ignoring-errors.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Ignoring Mixin Errors and Idempotent Actions"
+description: "Porter now supports ignoring the errors from a mixin. The az mixin takes advantage of this new feature to manage resource groups."
+date: "2022-01-07"
+authorname: "Carolyn Van Slyck"
+author: "@carolynvs"
+authorlink: "https://twitter.com/carolynvs"
+authorimage: "https://github.com/carolynvs.png"
+tags: ["mixins"]
+summary: |
+    Porter now supports ignoring the errors from a mixin. The az mixin takes advantage of this new feature to manage resource groups.
+---
+
+When is an error not really an error?
+Questions like this come up regularly when automating deployments.
+For example, when I repeat a script that creates a resource, I don't want it to error out because the resource already exists.
+A lot of command-line tools weren't designed in a way that works well with bundles.
+An installation could fail halfway through and need to be repeated.
+This is why we encourage bundle authors to use custom mixins, instead of just the exec mixin, because mixins are designed to work with bundles and give you **idemponent** behavior.
+Mixins should do what's necessary to match the desired state in the bundle and handle recoverable errors.
+
+Since we can't have a custom mixin for every tool and situation though, the exec mixin just got smarter.
+You can now handle errors directly from the exec mixin without having to fall back to writing scripts.
+
+Let's take a look at a few different ways that you can handle errors from the exec mixin.
+
+## Ignore All Errors
+
+Sometimes you want to run a command but don't really care if it fails.
+Now I won't tell you what to do in production, but when debugging a bundle, this can be handy.
+
+The snippet below will run a command, and Porter will ignore any errors returned by the command, and continue executing the next step in the action.
+
+```yaml
+install:
+  - exec:
+      description: "This may not work but such is life"
+      command: ./buy-me-coffee.sh
+      ignoreError:
+        all: true # Ignore any errors from this command
+```
+
+## Ignore Exit Codes
+
+Sometimes you get lucky, and the command that you are using has well-defined exit codes.
+In the example below, the made-up thing command returns 2 when the resource already exists.
```

Comment:
For the following example, the behavior of thing responds with the exit code [2] when the resource already exists. 

yaml configuration. 

In this example, since you don't care about the implied error as the desired outcome is the existence of resource, you configure porter to ignore the [2] exit code.